upup/models/cloudup/resources/addons/storage-aws.addons.k8s.io/v1.15.0.yaml

```
@@ -14,13 +14,28 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: gp2
+  labels:
+    k8s-addon: storage-aws.addons.k8s.io
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+
+---
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: gp2-kops
```

Maybe add purpose instead of the generic `kops` keyword?
```suggestion
  name: gp2-wait
```

---

That's another good option that avoids my prefix vs suffix nit-pick :-)

---

I think the purpose bit makes it too specific since I am also suggesting we set some other non-default settings